src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerRemoveImage.groovy

```diff
@@ -15,10 +15,24 @@
  */
 package com.bmuschko.gradle.docker.tasks.image
 
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.Optional
+
 class DockerRemoveImage extends DockerExistingImage {
+
+    @Input
+    @Optional
+    Boolean force
+
     @Override
     void runRemoteCommand(dockerClient) {
         logger.quiet "Removing image with ID '${getImageId()}'."
-        dockerClient.removeImageCmd(getImageId()).exec()
+        def removeImageCmd = dockerClient.removeImageCmd(getImageId())
+        
+        if(getForce()) {
+            removeImageCmd.withForce(getForce())
+        }
+        
+        removeImageCmd.exec()
```

Comment:
In case of `boolean` flags I think we can collapse this into one line:
````groovy
dockerClient.removeImageCmd(getImageId()).withForce(getForce()).exec()
````